### PR TITLE
Add Adrian Cole to the codeowners for the Brave module

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Comment by minwoox** [comment]
Let me contact the organization owner to give the write permission to @codefromthecrypt.

**Comment by codefromthecrypt** [comment]
happy to be here!

**Review by ikhoon — APPROVED** [review]
❤️❤️

**Review by jrhee17 — APPROVED** [review]
Note: there is also a project based on github actions in case granting write permission becomes an issue:
https://github.com/dyladan/component-owners

**Comment by codefromthecrypt** [comment]
aw man.. I broke the build!

**Comment by minwoox** [comment]
> there is also a project based on github actions in case granting write permission becomes an issue:

I didn't know about it. Let me try that, thanks!

File: .github/component-owners.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+components:
+  /:
```
Comment (minwoox) [inline .github/component-owners.yml:2]: `/` is used for all:
https://github.com/dyladan/component-owners/blob/main/src/utils.ts#L51

**Comment by minwoox** [comment]
Thanks! Let me try this. 😉

**Pull request merged**